modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -201,15 +201,17 @@
             expect(page).to have_test_selector('storage-provider-credentials-instructions',
                                                text: 'Copy these values from Nextcloud Administration / OpenProject.')
 
-            # With null values, submit button should be disabled
+            # With null values, form should render inline errors
             expect(page).to have_css('#oauth_client_client_id', value: '')
             expect(page).to have_css('#oauth_client_client_secret', value: '')
-            expect(find_test_selector('storage-oauth-client-submit-button')).to be_disabled
+            click_button 'Save and continue'
+
+            expect(page).to have_text("Client can't be blank.")
```

Comment:
🟡 nit: shouldn't the validation text say: "Client ID can't be blank."?

Reply:
Good eye, this is a primer side effect of any form object with an `_id` prefix in the name- I have not be able to find a workaround but can log a bug ticket for investigation. Any ideas @HDinger @jonasjabari ?

Reply:
looks good!